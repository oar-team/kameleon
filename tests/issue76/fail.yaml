global:
    arch: x86_64
    iso_arch: $${arch}
    iso_filename: debian-jessie-$${iso_arch}-live.iso 

bootstrap:
    - testbootstrap:
      - toto: $${arch}
      - step:
        - exec_local: echo $iso_filename
