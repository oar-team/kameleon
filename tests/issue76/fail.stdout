Warning : variable $$arch is not enclosed with braces, which may cause errors. Please prefer using $${arch}.
Warning : variable $$arch-live is not enclosed with braces, which may cause errors. Please prefer using $${arch-live}.
Error : /home/neyron/scm/OAR/kameleon/tests/issue76/fail.yaml: variable $$arch-live not found in local or global
